src/Parser/PhpDocParser.php

```diff
@@ -178,6 +178,9 @@ public function parseTagValue(TokenIterator $tokens, string $tag): Ast\PhpDoc\Ph
 				case '@template-covariant':
 				case '@phpstan-template-covariant':
 				case '@psalm-template-covariant':
+				case '@template-contravariant':
+				case '@phpstan-template-contravariant':
+				case '@psalm-template-contravariant':
```

Comment:
Psalm doesn't currently support @template-contravariant tags either, so I'm unsure whether this should be included.

Support [would be welcomed](https://github.com/vimeo/psalm/issues/5252), so I'd rather keep this.

Reply:
Yes, there's no harm in having `@psalm-template-contravariant` here :)